e2e-tests/IIoTPlatform-E2E-Tests/IIoTPlatform-E2E-Tests.csproj

```diff
@@ -21,4 +23,8 @@
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\common\src\Microsoft.Azure.IIoT.Core\src\Microsoft.Azure.IIoT.Core.csproj" />
```

Comment:
><ProjectReference Include="..\..\common\src\Microsoft.Azure.IIoT.Core\src\Microsoft.Azure.IIoT.Core.csproj" /> [](start = 4, length = 110)

Please do not reference Industrial IoT Platform assemblies, the Test should be as independent as possible.

Reply:
Removed reference to `Microsoft.Azure.IIoT.Core` project.